src/api/identity.rs

```diff
@@ -812,11 +989,137 @@ struct ConnectData {
     two_factor_remember: Option<i32>,
     #[field(name = uncased("authrequest"))]
     auth_request: Option<AuthRequestId>,
+    // Needed for authorization code
+    #[form(field = uncased("code"))]
```

Comment:
This is different than the others, is there a reason for that? It seems to be building either way which is a bit weird, I think
```suggestion
    #[field(name = uncased("code"))]
```